.github/workflows/upload_all.yml

```diff
@@ -86,12 +97,10 @@ jobs:
         with:
           serviceAccountJsonPlainText: ${{ secrets.POLYPOD_ANDROID_STORE_SERVICE_ACCOUNT_JSON }}
           packageName: coop.polypoly.polypod
-          releaseName: ${{ steps.vars.outputs.release_name }}
+          releaseName: ${{ steps.determine-release-name.outputs.release_name }}
           releaseFiles: ./platform/android/app/build/outputs/apk/release/app-release.apk
           track: internal
-          status: draft
-          # whatsNewDirectory: distribution/whatsnew
-          # mappingFile:
```

Comment:
I thought these were needed when publishing as non-draft - but turns out they are not, so I just got rid of the comments.